# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserParent.h
# Commit: 00d83f1d02e0
# Full Hash: 00d83f1d02e015735d580045524eceeeccdc4e28
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 04:41:59
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 1 - Allow calling BrowserParent::InitRendering multiple times, and remove RenderFrame dependency from nsFrameLoader. r=kats
#   
#   This cleanup will simplify refactoring nsFrameLoader later.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31429
# ==============================================================================

diff -r bdd76368ddc1 -r 00d83f1d02e0 dom/ipc/BrowserParent.h
--- a/dom/ipc/BrowserParent.h	Wed May 22 18:37:34 2019 +0300
+++ b/dom/ipc/BrowserParent.h	Wed Apr 24 22:36:27 2019 -0500
@@ -463,10 +463,7 @@
   void InitRendering();
   void MaybeShowFrame();
 
-  // XXX/cjones: it's not clear what we gain by hiding these
-  // message-sending functions under a layer of indirection and
-  // eating the return values
-  void Show(const ScreenIntSize& aSize, bool aParentIsActive);
+  bool Show(const ScreenIntSize& aSize, bool aParentIsActive);
 
   void UpdateDimensions(const nsIntRect& aRect, const ScreenIntSize& aSize);
 
